source/apiVolontaria/volunteer/models.py

```diff
@@ -351,3 +374,41 @@ def duration(self):
             return timedelta(minutes=self.presence_duration_minutes)
         else:
             return self.event.duration
+
+    def generate_email_invitation(self):
+        calendar = self.event.generate_ics()
+
+        if settings.CONSTANT['EMAIL_SERVICE'] is True:
+
+            email_to = self.user.email
+
+            # Debug Mode checking if we have an email in DEBUG_EMAIL_TO
+            # or disable the sending of email
+            if settings.DEBUG:
+                try:
+                    email_to = settings.DEBUG_EMAIL_TO
+                except AttributeError:
+                    email_to = None
+
+            msg = EmailMultiAlternatives(
+                subject=None,
+                body=None,
+                to=[email_to, ],
+            )
+
+            msg.attach('nousrire.ics', calendar.decode(), 'text/calendar')
```

Comment:
Please keep name generic and understandable. `nousrire.ics` is not really easy to understand and is not generic, maybe can we use `invitation.ics`.

Reply:
Good point !